src/main/java/com/wolox/training/controllers/UserController.java

```diff
@@ -24,6 +25,7 @@
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 
+import java.security.Principal;
```

Comment:
Delete unnecessary changes to avoid modifying files that are not involved in the solution